addon/components/select-component.js

```diff
@@ -145,7 +144,7 @@ export default Ember.Component.extend(
     } else {
       return this.get('originalItemViewClass');
     }
-  }).property('showTooltip'),
```

Comment:
So, we definitely disabled prototype extensions in Iverson. I could have sworn we updated Ember Widgets as well, unsure how this is still here